android/walletlib/src/main/java/com/solana/mobilewalletadapter/walletlib/provider/PowerConfigProvider.java

```diff
@@ -0,0 +1,5 @@
+package com.solana.mobilewalletadapter.walletlib.provider;
+
+public interface PowerConfigProvider {
```

Comment:
Proposal: move this class to `com.solana.mobilewalletadapter.walletlib.scenario`, and make it package-scoped. That way, we limit its use to only test code.